scripts/generate-e2e-test-matrix.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e
+
+# Find all tests
+cd e2e/interchaintestv8
```

Comment:
This was the pretties I was able to make this in bash... Could've made it nicer in go or something probably. But as long as it works 🤷